features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/SessionReplayRecorder.kt

```diff
@@ -104,8 +109,23 @@ internal class SessionReplayRecorder : OnWindowRefreshedCallback, Recorder {
         this.recordedDataQueueHandler = RecordedDataQueueHandler(
             processor = processor,
             rumContextDataHandler = rumContextDataHandler,
-            timeProvider = timeProvider,
-            internalLogger = internalLogger
+            internalLogger = internalLogger,
+
+            /**
+             * TODO RUMM-0000 consider change to LoggingThreadPoolExecutor once V2 is merged.
```

Comment:
I think it is time to do it: it is a simple change and can be made in another PR